Event/Subscriber/DoctrineORMSubscriber.php

```diff
@@ -86,7 +87,7 @@ public function filterEntity(GetFilterConditionEvent $event)
 
             if ($dqlFrom = $event->getQueryBuilder()->getDQLPart('from')) {
                 $rootPart = reset($dqlFrom);
-                $fieldName = ltrim($event->getField(), $rootPart->getAlias() . '.');
+                $fieldName = \str_replace(\sprintf('%s.', $rootPart->getAlias()), null, $event->getField());
```

Comment:
str_replace works better than ltrim in some case?

Reply:
Using **ltrim** method is not correct here. **ltrim** replace all listing characters and not word.

Example, 2 entities : 

1. **Document** with alias **document**
2. **Category** with alias **category**

```php
dump($event->getField()); // document.category
dump($rootPart->getAlias() . '.'); // document.

// Before (wrong)
dump(ltrim($event->getField(), $rootPart->getAlias() . '.')); // ategory

// After (good)
dump(\str_replace(\sprintf('%s.', $rootPart->getAlias()), null, $event->getField())); // category
```